examples/struct_with_subargs.rs

```diff
@@ -11,9 +11,14 @@ use interactive_clap::{ResultFromCli, ToCliArgs};
 #[derive(Debug, Clone, interactive_clap::InteractiveClap)]
 struct Account {
     /// Change SocialDb prefix
+    ///
+    /// It's a paraghraph, describing, this argument usage in more detail
+    /// than just the headline
     #[interactive_clap(long)]
     #[interactive_clap(skip_interactive_input)]
+    #[interactive_clap(verbatim_doc_comment)]
```

Comment:
`verbatim` is so weird word. Why `clap` hasn't used something more simple. Just a comment ignore

Reply:
verbatim (not comparable)

Word for word; in exactly the same words as were used originally.
I have copied his speech verbatim, so this is exactly what he said, word for word.

It was used in a sense, that this attr preserves original whitespace

Reply:
Yeah I goggled it, weird word anyway